package browsernode.ljstream_messages;

import "js/javascript_package.proto";
option (javascript_package) = "ljstream";

message NewPost {
	required string title = 1;
	required string url = 2;
	required string body = 3;
	required int32 num_images = 4;
	required string lang = 5;
}

message SetPreferences {
	required bool include_russian_posts = 1;
}
